ra/registration-authority.go

```diff
@@ -428,49 +357,35 @@ func (ra *RegistrationAuthorityImpl) NewCertificate(req core.CertificateRequest,
 	err = ra.checkLimits(names, registration.ID)
 	if err != nil {
 		logEvent.Error = err.Error()
-		return emptyCert, err
+		return emptyCertRequest, err
 	}
 
-	err = ra.checkAuthorizations(names, &registration)
+	earliestExpiration, err := ra.checkAuthorizations(names, &registration)
 	if err != nil {
 		logEvent.Error = err.Error()
-		return emptyCert, err
+		return emptyCertRequest, err
 	}
+	req.Expires = earliestExpiration
```

Comment:
I know we don't like extraneous documentation, but here I think it'd be worthwhile to explain how the `Expires` comes into play in issuance.